pkg/metrics/metrics.go

```diff
@@ -121,6 +122,31 @@ The label 'result' can have the following values:
 For a ClusterQueue, the metric only reports a value of 1 for one of the statuses.`,
 		}, []string{"cluster_queue", "status"},
 	)
+
+	// Optional cluster queue metrics
+	ClusterQueueResourceUsage = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: constants.KueueName,
+			Name:      "cluster_queue_resource_usage",
+			Help:      `Reports the cluster_queue's total resource usage within all the flavors`,
+		}, []string{"cohort", "cluster_queue", "flavor", "resource"},
+	)
+
+	ClusterQueueResourceNominalQuota = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: constants.KueueName,
+			Name:      "cluster_queue_nominal_quota",
```

Comment:
What's the rational for recording this is metrics, if it's something that the users input in the spec?

Reply:
That could also be done from the borrowing metric.
Fewer metrics is preferred for lower memory usage.

Reply:
not 100%, you can extract the nominal quota if the resource has a borrowing limit and the usage is more than that. So you can only say if you are using all the nominal quota or not. I think that being able to know the % of usage out of nominal quota and borrowing is important from an  UX POV.